README.rst

```diff
@@ -217,6 +217,49 @@ The same functionality can be applied to APIs built with Flask-RESTful
     $ curl localhost:5000 -H "Accept: application/vnd.your_vendor.v3"
     Goodbye cruel world.
 
+Works with `Flask-RESTPlus <https://pypi.org/project/flask-restplus/>`_ Resources
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The same functionality can be applied to APIs built with Flask-RESTPlus
+
+.. code:: python
+
+    from flask import Flask, jsonify
+    from flask_accept import accept
+    from flask_restplus import Resource, Api
+    app = Flask(__name__)
+    api = Api(app)
+
+
+    class HelloWorldResource(Resource):
+        @accept('application/vnd.your_vendor.v1', 'application/vnd.your_vendor.v2')
+        def get():
+        """
+            The doc string showing in swagger
+        """
```

Comment:
Could you indent the docstring quotes here to adhere to [PEP 257](https://www.python.org/dev/peps/pep-0257/)?